main_server/local_server/bmlt.js

```diff
@@ -16,22 +16,25 @@ var militaryToStandard = function(value) {
             return value;
         }
         else {
-            if(value.length == 8){ //If value is the expected length for military time then process to standard time.
-                var hour = value.substring ( 0,2 ); //Extract hour
-                var minutes = value.substring ( 3,5 ); //Extract minutes
-                var identifier = 'AM'; //Initialize AM PM identifier
+            if (value.length == 8) { //If value is the expected length for military time then process to standard time.
+                value = value.split(':'); // convert to array
```

Comment:
One other suggestion is to use a different variable other than `value` for assignment.  It's generally not a good idea to overwrite variables.  You lose access to the original value, and you make yourself prone to other such errors potentially.

Reply:
makes sense